demo/src/app/app.module.ts

```diff
@@ -2,7 +2,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
-import { BootstrapModalModule } from 'angularx-bootstrap-modal';
+import { DialogModalModule } from 'angularx-bootstrap-modal';
```

Comment:
#4 Might as well get some other breaking changes into this single big change. Module name renamed